plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,11 @@
+#! bash oh-my-bash.module
+
+function _omb_plugin_sudo_add_sudo {
+  if [[ -z $READLINE_LINE ]]; then
+    READLINE_LINE="$(fc -ln -1 | command sed 's/^[[:space:]]\{1,\}//')"
+  fi
+  READLINE_LINE="sudo $READLINE_LINE"
+  ((READLINE_POINT += 5))
+}
+
+bind -x '"\e\e": _omb_plugin_sudo_add_sudo'
```

Comment:
What I discuss above is the three keybindings in [ohmyzsh sudo plugin L106-108](https://github.com/ohmyzsh/ohmyzsh/blob/d47e1d65f66f9bb2e7a96ba58797b33f0e91a623/plugins/sudo/sudo.plugin.zsh#L106-L108). What I suggest is the following change:

```suggestion
bind -m emacs -x '"\e\e": _omb_plugin_sudo_add_sudo'
bind -m vi-insert -x '"\e\e": _omb_plugin_sudo_add_sudo'
bind -m vi-command -x '"\e\e": _omb_plugin_sudo_add_sudo'
```